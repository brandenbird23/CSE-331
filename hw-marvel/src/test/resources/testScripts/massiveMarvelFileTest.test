# Here is a few tests from Ameya Agrawal Discussion post

LoadGraph marvel marvel.csv

# Should have 2 length path
FindPath marvel IRON-MAN/TONY-STARK KANGAROO-II

# Should have 3 length path
FindPath marvel GOOM MR.-FANTASTIC/REED-R

# Should have 3 length path
FindPath marvel SHA-SHAN SQUIRREL-GIRL

# Should Return No Path Found
FindPath marvel GOOM HOFFMAN