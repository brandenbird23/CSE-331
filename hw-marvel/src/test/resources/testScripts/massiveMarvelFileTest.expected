# Here is a few tests from Ameya Agrawal Discussion post

loaded graph marvel

# Should have 2 length path
path from IRON-MAN/TONY-STARK to KANGAROO-II:
IRON-MAN/TONY-STARK to ABSORBING-MAN/CARL-C via A-183
ABSORBING-MAN/CARL-C to KANGAROO-II via W2-164

# Should have 3 length path
path from GOOM to MR.-FANTASTIC/REED-R:
GOOM to VENUS/APHRODITE/VICT via M/VS-1/3
VENUS/APHRODITE/VICT to ANGEL/WARREN-KENNETH via CHAMP-1
ANGEL/WARREN-KENNETH to MR.-FANTASTIC/REED-R via A-3

# Should have 3 length path
path from SHA-SHAN to SQUIRREL-GIRL:
SHA-SHAN to ANGEL/WARREN-KENNETH via PPTSS-18
ANGEL/WARREN-KENNETH to DR.-DOOM/VICTOR-VON via CHAMP-16
DR.-DOOM/VICTOR-VON to SQUIRREL-GIRL via M/SH3-8/3

# Should Return No Path Found
path from GOOM to HOFFMAN:
no path found